packages/sdk/index.js

```diff
@@ -1,6 +1,6 @@
-import createMiddleware from './middleware';
-import * as actions from './actions';
-import SdkGet from './components/sdk-get';
+import createMiddleware from './src/middleware';
+import * as actions from './src/actions';
+import SdkGet from './src/components/sdk-get';
 
 const Sdk = {
```

Comment:
Because `test-utils` should not be part of the main bundle right? Isn't that what we agreed on?

Reply:
@emmenko exactly

Reply:
Yeah that was my understanding of the intent too. I was just assuming tree shaking would do its part. Anyway, it makes sense to not mix up app and test utils code I suppose. Thanks.

Reply:
@tdeekens treeshaking is hard and we already have problems with it. I would rather do it like this 🤷‍♂️

Reply:
Yes I agree it being hard and this being a totally valid option to get around to not relying on it. However it would still be cool to know if it would work in this case as it is rather simplistic and would simplify folder structure and build system.